### DISCUSSION TITLE
Is in the roadmap to support plda metric?Probabilistic Linear Discriminant Analysis

### DISCUSSION THREAD

**Discussion by sandraplato1234** [milvus-io/milvus#15444] [Q&A and General discussion]
Hi 
Is in the roadmap to support plda metric?Probabilistic Linear Discriminant Analysis
Thanks in advance

**Comment by xiaofan-luan** [comment]
Not in the roadmap yet. We are considering implement a udf for data preprocessing but it's not done yet~
If you are interested feel free to add a MEP (https://wiki.lfaidata.foundation/pages/viewpage.action?pageId=43287103) to add a preprocessing stage and we can do plda there.

Also if you are using faiss index, maybe simply use PCA for dimensionality reduction
